### PR TITLE
Update index.html with signed interest to join the cause for OpenTF

### PR DESCRIPTION
**Pull request by fahadahammed**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -2423,11 +2423,16 @@ <h3>Co-signed</h3>
               <td>Individual</td>
               <td>Development; open-source community efforts</td>
             </tr>
+		<tr>
```
Comment (edizsuleyman) [inline index.html:2426]: Please make sure to be consistent with the alignments of `<tr>` tags

File: index.html [hunk 2]
```diff
@@ -2423,11 +2423,16 @@ <h3>Co-signed</h3>
...
+              <td><a href="https://github.com/fahadahammed">Fahad Ahammed</a></td>
+              <td>Individual</td>
+              <td>Testing;Documentation;open-source community efforts</td>
```
Comment (edizsuleyman) [inline index.html:2429]: Make sure to add spacing in the words here:
`<td>Testing; Documentation; open-source community efforts</td>`

File: index.html (hunk 1 above)
Comment (edizsuleyman) [inline index.html:2426]: Please make sure to be consistent with the alignments of` <tr>` tags

**Review by edizsuleyman — CHANGES_REQUESTED** [review]

**Comment by flavius-dinu** [comment]
Added in https://github.com/opentffoundation/manifesto/pull/533

**Pull request closed without merging**